commands/board/list.go

```diff
@@ -105,15 +105,19 @@ func List(instanceID int32) ([]*rpc.DetectedPort, error) {
 	m.Lock()
 	defer m.Unlock()
 
+	tags := map[string]string{}
+	// Use defer func() to evaluate tags map when function returns
+	defer func() { stats.Incr("board.list", stats.M(tags)...) }()
+
 	pm := commands.GetPackageManager(instanceID)
 	if pm == nil {
-		stats.Incr("board.list", stats.T("success", "false"))
+		tags["success"] = "false"
```

Comment:
I think we can push this further with named return params.

Reply:
Can you give me an example of that?